README.MD

```diff
@@ -124,6 +126,21 @@ With the default configuration, all commits on develop will be pushed to Firebas
 To also make these available to a test group, you can enable the `groups` parameter in the
 [firebase yml](.github/workflows/firebase-app-distribution.yml) file.
 
+#### Play Store deployment
+
+We use the [upload-google-play github action](https://github.com/r0adkll/upload-google-play) in our github actions setup to automatically upload release builds to the Play Store Internal Test track.
+
+To enable this for your own app, you need to:
+
+1. Create a service account in Google Cloud Console with access to the Google Play Developer API
+2. Download the service account JSON key file
+3. Add the content of the JSON file to your Github repository secrets as `PLAYSTORE_SERVICE_ACCOUNT_JSON`
+4. Update the `packageName` in the [playstore-internal.yml](.github/workflows/playstore-internal.yml) file to match your app's package name
+
+More details on creating a service account can be found in the [Google Play Android Developer API documentation](https://developers.google.com/android-publisher/getting_started).
```

Comment:
Use this as the documentation url instead: https://github.com/r0adkll/upload-google-play?tab=readme-ov-file#configure-access-via-service-account